android/app/src/main/java/io/imagekit/imagekitdemo/MainActivity.kt

```diff
@@ -0,0 +1,48 @@
+package io.imagekit.imagekitdemo
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.facebook.drawee.backends.pipeline.Fresco
+import com.imagekit.android.ImageKit
+import com.imagekit.android.entity.TransformationPosition
+import io.imagekit.imagekitdemo.databinding.ActivityMainBinding
+
+class MainActivity : AppCompatActivity() {
+    private var binding: ActivityMainBinding? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding?.root)
+
+        ImageKit.init(
+            context = applicationContext,
+            publicKey = "IK_PUBLIC_KEY",
+            urlEndpoint = "https://ik.imagekit.io/IMAGEKIT_ID",
```

Comment:
We should move these to a `constants.kt` file with proper comments and use the values everywhere we need

For `urlendpoint`, the default value should be `https://ik.imagekit.io/IMAGEKIT_ID` and have a comment saying `Replace with ImageKit URL endpoint (or CNAME) from ImageKit Dashboard (https://imagekit.io/dashboard/url-endpoints)`.

We can have similar comments for other values also

And in the readme, we can instruct the user to directly change the values there

Reply:
Done. Please check